packaging/systemd/microshift-containerized.service

```
@@ -13,8 +13,10 @@ RequiresMountsFor=%t/containers
 Environment=PODMAN_SYSTEMD_UNIT=%n
 Restart=on-failure
 TimeoutStopSec=70
+ExecStartPre=/bin/bash /usr/bin/microshift-containerized-configure-selinux # not autogenerated
```

```suggestion
ExecStartPre=/bin/bash /usr/bin/microshift-containerized-configure-selinux # not autogenerated
``` 

We should not do this, it's a lengthy process. ( installing, compiling, etc...)

Let's assume that people will download the rpms for microshift via dnf, which will include both microshift-containerized + the selinux modules compiled.

We can keep the script available just in case.
```suggestion
```

---

https://github.com/redhat-et/microshift/blob/main/packaging/rpm/microshift.spec#L226